### DISCUSSION TITLE
THE GOOD, THE BAD, AND THE UGLY OF MOBY/BUILDKIT (PART 1) | 高策

### DISCUSSION THREAD

**Discussion by giscus** [gaocegege/blog-comments#8] [General]
# THE GOOD, THE BAD, AND THE UGLY OF MOBY/BUILDKIT (PART 1) | 高策

江湖小虾米

http://gaocegege.com/Blog/kubernetes/buildkit

**Comment by yeya24** [comment]
gaocegege 这个邮箱是不是有问题？看了下你 GH profile 里面写的是 cegao@tensorchord.ai 不是 io

**Reply by gaocegege** [reply]
是的，我改一下
